common/genes/03_genes_accessories_beards.txt

```diff
@@ -283,5 +283,14 @@ accessory_genes = {
 				1 = empty
 			}
 		}
+		uther_beard = {
```

Comment:
The question, do you plan to force this beard on Uther using a portrait modifier?

Reply:
> 
> 
> The question, do you plan to force this beard on Uther using a portrait modifier?

Yes, I will make a portrait modifier similar to what Lothar has so Uther's beard changes to this one when he's 40 or so.